retworkx/__init__.py

```diff
@@ -605,9 +605,9 @@ def _graph_dijkstra_shortest_path(
 
 @functools.singledispatch
 def all_pairs_dijkstra_shortest_paths(graph, edge_cost_fn):
-    """Find the shortest path from all nodes
+    """Find the shortest path from all nodes in a graph
 
-    This function will generate the shortest path from all nodes int the graph
+    This function will generate the shortest path from all nodes in the graph
```

Comment:
Should we also mention what happens if there's a tie between multiple shortest paths?

Reply:
Yeah, that's probably useful given the questions in #546. It's not the easiest to describe though because it depends on the exact graph. It will return one of the paths with a bias towards edges added more recently as that's the inner loop in the dijkstra implementation.